content/tv/springone-tour/0017.md

```diff
@@ -13,15 +13,68 @@ aliases: ["/tv/springone-tour/17","/s1tour-emea-booternetes-II-the-yaml-strkes-b
 contact: springone@vmware.com
 contact_text: the SpringOne Tour Team
 speakers:
+  - Michael Coté
+  - Olga Maciaszek-Sharma
+  - Marcin Grzejszczak
+  - Jakub Pilimon
+  - Jonatan Ivanov
+  - Nenad Momcilovic 
+  - Timo Salm
+  - Ivan Tarin
 guests: []
-Date: "2021-06-15T10:00:00-07:00"
+Date: "2021-06-15T00:00:00-04:15"
```

Comment:
Is this right? This is only the start time so I think it should be `Date: "2021-06-15T10:00:00-07:00"` as it was, which is 10am PDT. If we want to specify that it runs 4 hours or whatever we can add `minutes: 240`.